open-metadata-implementation/user-interfaces/presentation-server/README.md

```diff
@@ -38,14 +38,6 @@ the types of [Egeria OMAG Servers](../../admin-services/docs/concepts/omag-serve
  * Developers can specify the environment variable(s) in a .env file, by copying [.env_sample](nodejs/.env_sample) to a file
  called .env in the same folder and amend the values as required.
  * Once the .env file is in place, build and start the server using `npm start`. Use other npm parameters as specified in [package.json](nodejs/package.json).  
-
-## FOR NEW DEVELOPMENT (as of 8/2/2020)
- * Currently, the UI is not optimized to run in development mode. Developers must rebuild the static files every time a change is made.
```

Comment:
You need to update the .gitignore in the root folder so that all the generated code from the build is not added into git. I suggest copying the .gitignore part for the existing presentation-server and changing for the cra content.